File: examples/python/core/invoke/parent-workflow.py
@durable_execution
def parent_workflow(event: dict, context: DurableContext) -> dict:
    """Parent workflow that coordinates sub-workflows."""
    project_id = event["project_id"]
    
    # Invoke sub-workflow for data collection
    data = context.invoke(
        function_name="collect-data-workflow",
        payload={"project_id": project_id},
        name="collect_data",
    )
    
    # Invoke sub-workflow for data processing
    processed = context.invoke(
        function_name="process-data-workflow",
        payload=data,
        name="process_data",
    )
    
    # Invoke sub-workflow for reporting
    report = context.invoke(
        function_name="generate-report-workflow",
        payload=processed,
        name="generate_report",
    )
    
    return report

File: examples/java/core/invoke/parent-workflow.java
public class ParentWorkflowHandler extends DurableHandler<Map<String, Object>, Map<String, Object>> {

    @Override
    public Map<String, Object> handleRequest(Map<String, Object> event, DurableContext ctx) {
        String projectId = (String) event.get("project_id");

        // Invoke sub-workflow for data collection
        var data = ctx.invoke(
            "collect_data",
            "collect-data-workflow",
            Map.of("project_id", projectId),
            Map.class
        );

        // Invoke sub-workflow for data processing
        var processed = ctx.invoke(
            "process_data",
            "process-data-workflow",
            data,
            Map.class
        );

        // Invoke sub-workflow for reporting
        var report = ctx.invoke(
            "generate_report",
            "generate-report-workflow",
            processed,
            Map.class
        );

        return report;
    }
}
